README.md

```diff
@@ -136,6 +137,14 @@ Please be aware that the demos may exhibit significant accessibility issues, suc
 
 ---
 
+## <a id="social-icons"></a>Social_media_icons
+
+[<img src="images/icons.png" height="230" title="Pendulum Animation Demo">]
```

Comment:
change the title of the img tag here @Akrati00

Reply:
> change the title of the img tag here @Akrati00

done